src/checker.py

```diff
@@ -77,7 +80,12 @@ def _get_module_data_from_json(self, path, json_data):
             if isinstance(module, str):
                 module_path = os.path.join(os.path.dirname(self._manifest),
                                            module)
-                module = self._read_manifest(module_path)
+                try:
+                    module = self._read_manifest(module_path)
+                except GLib.Error as err:
+                    if err.matches(GLib.quark_from_string('g-file-error-quark'), 4):
```

Comment:
```suggestion
                    if err.matches(GLib.quark_from_string('g-file-error-quark'), GLib.FileError.NOENT):
```

but I think that if the manifest is a YAML file, this will raise `FileNotFoundError` instead. You could either handle both here, or (maybe neater) make `def read_json_manifest(manifest_path):` convert from the GLib flavour to the Python flavour, or replace:

```
    parser.load_from_file(manifest_path)
```

with
```
with open(manifest_path, "r") as f:
    parser.load_from_data(f.read())
```